### PR TITLE
created sheet 1800: el ocaso de la humanidad

### PR DESCRIPTION
**Pull request by Hornwar**
This is a character sheet for the spanish rol game 1800: el ocaso de la humanidad (the game will be released in english next year). This sheet has been created as per the standards required from roll20.

## Roll20 Requests

Comments are very helpful for reviewing the code changes. Please answer the relevant questions below in your comment.

- [ ] Does the pull request title have the sheet name(s)? Include each sheet name.
- [ ] Is this a bug fix?
- [ ] Does this add functional enhancements (new features or extending existing features) ?
- [ ] Does this add or change functional aesthetics (such as layout or color scheme) ? 
- [ ] If changing or removing attributes, what steps have you taken, if any, to preserve player data ?
- [x] If this is a new sheet, did you follow [Building Character Sheets standards](https://wiki.roll20.net/Building_Character_Sheets#Roll20_Character_Sheets_Repository) ?

If you do not know English. Please leave a comment in your native language.

### CODE REVIEW

**Comment by Anduh** [comment]
I'd strongly recommend you to change the sheet to not use `<table>` elements for styling. As stated in the [Good Code](https://wiki.roll20.net/Building_Character_Sheets#2._Good_Code) section in the Character Sheet Standards:

> **Proper HTML Syntax.** Proper HTML syntax is encouraged to increase accessibility and make the code maintainable for community contributions. As a general standard a `<table>` element should only be used for tabular data. The `<table>` shall not be used for layouts. All new sheets are expected to use proper containers elements such as `<div>` and `<span>` elements. Your HTML file must not use `<⁠head>` or `<⁠body>` tag, or your character sheet may not load in the virtual tabletop. 

regarding the `sheet.json` files "instruction"-section, use `\n` istead of linebreaks for formatting the text. Otherwise you can use Markdown syntax for for matting it, including hyperlinks. [Example](https://wiki.roll20.net/Building_Character_Sheets#2._Good_Code)

File: 1800 El ocaso de la humanidad/1800 el ocaso de la humanidad.css [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+}
+.charsheet input[type="number"].sheet-medium::-webkit-inner-spin-button {
+    -webkit-appearance: none;
+}
+.charsheet select.sheet-short{
+    display: inline-block;
+    margin: 0px 0px;
+    width: 4.1em;
+}
+.charsheet input[type="checkbox"].left {
```
[the remark this answers is not in the document]
Comment (Hornwar) [reply 1800 El ocaso de la humanidad/1800 el ocaso de la humanidad.css]: Thanks for the reply. To be honest, this is the first time I ever written any code, I basically copied the body of another existing character sheet (wich uses tables all around), changed the top logo and the background and the names of the fields. Any sugestion on what elements i should use to not screw up the entire character sheet?

[the remark this answers is not in the document]
Comment (Anduh) [reply 1800 El ocaso de la humanidad/1800 el ocaso de la humanidad.css]: You did great on having all the files needed and structured the PR well, so this is better than most submissions, and probably better than my first submission.

The [Building Sheets wiki article](https://wiki.roll20.net/Building_Character_Sheets) have many tips and links that helps with character sheet creation, including [Character sheet template examples](https://wiki.roll20.net/Building_Character_Sheets#See_Also). All these are updated and follows Roll20 minimum standards, and also includes suggestions how to structure sheets.

Yes, older character sheets use tables, but it's a very old method that is hard to make changes to and change how it looks, so metods like [CSS Flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) or [CSS Grid ](https://css-tricks.com/snippets/css/complete-guide-grid/) are the most popular.

[the remark this answers is not in the document]
Comment (Hornwar) [reply 1800 El ocaso de la humanidad/1800 el ocaso de la humanidad.css]: I understand then, that I will have to start the sheet almost from zero. Unfortunately, I did this one  on my holiday time, so there is no chance I will get this done any time soon. As there isn't any other option for this game, would be possible to push it forward until I get a better version?
I am sure players will be happier with this sheet that any at all.

**Comment by Anduh** [comment]
I'm just a community member, I don't decide anything.
This was just general feedback, and you'll get official response when the sheet gets reviewed.

File: 1800 El ocaso de la humanidad/1800 el ocaso de la humanidad.html [hunk 2]
```diff
@@ -0,0 +1,1504 @@
+<div>
+  <img src="https://i.imgur.com/ZVxL8ZG.png" width="800" height="800"  />
+ </div> 
+ <tr> <td colspan="10">&nbsp;</td>
+ </tr>
+<div class="sheet-maindiv">
+<div class="sheet-2colrow">
+   <div class="sheet-col">
+      <table>
```
Comment (clevett) [inline 1800 El ocaso de la humanidad/1800 el ocaso de la humanidad.html]: New sheets looking to be added to the repository are required to meet the [Building Character Sheet](https://wiki.roll20.net/Building_Character_Sheets) standards. In particular character sheets should not use `<table>` for layouts. Older sheets in the past have used this method but it is no longer accepted.

**Pull request merged**